src/styles/block/stars.css

```diff
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+
+  /* background-size: contain; */
```

Comment:
Remove comments everywhere

Reply:
When I add the property ".stars__star:last-child { margin-right: 0;}"  the test fails (npm t).